pages/api/feedContent/[id].ts

```diff
@@ -0,0 +1,144 @@
+import type {
+  FeedComment,
+  FeedItemContent,
+} from "../../../components/feed/FeedPage";
+
+const api_url =
+  "https://firestore.googleapis.com/v1/projects/dkms-spotify/databases/(default)/documents/feed_content";
+const base_url = "https://firestore.googleapis.com/v1/";
+
+type FeedCommentDocument = {
+  name: string;
+  fields: {
+    username: {
+      stringValue: string;
+    };
+    comment: {
+      stringValue: string;
+    };
+  };
+  createTime: Date;
+};
+
+type FeedCommentResponse = {
+  documents: FeedCommentDocument[];
+};
+
+type FeedContentDocument = {
+  name: string;
+  fields: {
+    username: { stringValue: string };
+    content: { stringValue: string };
+  };
+  createTime: Date;
+};
+
+type FeedContentResponse = {
+  documents: FeedContentDocument[];
+};
+
+const uniqueId = () => {
+  const dateString = Date.now().toString(36);
+  const randomness = Math.random().toString(36).substr(2);
```

Comment:
```suggestion
  const randomness = Math.random().toString(36).substring(0,2);
```

We should probably change how IDs are generated but this works for now. Just use substring bcs substr is deprecated